docs/platypus2/go.mod

```diff
@@ -1,6 +1,9 @@
 module github.com/golingon/lingoneks
 
-go 1.22.2
+go 1.23.0
```

Comment:
```suggestion
go 1.23
```

Reply:
I cannot get this to work. `go mod tidy` will always add the patch version.